plugins/phoenix/phoenix_main.ml

```diff
@@ -17,56 +16,43 @@ let run options project =
   let dest = Phoenix.store () in
   printf "Stored data to %s@." dest
 
-let cfg_format : 'a list Term.t =
-  Arg.(value & vflag_all [`with_name] [
-      `with_name, info ["labels-with-name"]
-        ~doc: "Put block name on graph labels";
-      `with_asm, info ["labels-with-asm"]
-        ~doc:"Put assembler instructions on graph labels";
-      `with_bil, info ["labels-with-bil"]
-        ~doc:"Put bil instructions on graph labels";
-    ])
+let label_with x y =
+  let name = sprintf "labels-with-%s" x in
+  let doc = sprintf "Put %s on graph labels" y in
+  Config.(flag name ~doc)
 
-let output_folder : _ Term.t =
-  let doc = "Output data into the specified folder" in
-  let default = "phoenix" in
-  Arg.(value & opt string default & info ["output-folder"] ~doc)
+let with_name = label_with "name" "block name"
+let with_asm  = label_with "asm"  "assembler instructions"
+let with_bil  = label_with "bil"  "bil instructions"
```

Comment:
Well, with the current `Config` interface then we cannot preserve this, and will have to extend our interface.